dapps/templates/boilerplate/embark.json

```diff
@@ -12,6 +12,13 @@
     "solc": "0.5.0"
   },
   "plugins": {
+    "embark-ipfs": {},
```

Comment:
Should what's added here and in `package.json` in this and the other templates also be added to the dapps in `dapps/tests`?

Reply:
added to dapp/tests as well